test/test_distributions.py

```diff
@@ -100,6 +100,12 @@ def __new__(cls, jax_dist, *params):
     T(dist.StudentT, 1., 1., 0.5),
     T(dist.StudentT, 2., np.array([1., 2.]), 2.),
     T(dist.StudentT, np.array([3, 5]), np.array([[1.], [2.]]), 2.),
+    T(dist.TruncatedCauchy, -1., 0., 1.),
```

Comment:
Since these distributions have corresponding scipy implementations available, can we add these to `_DIST_MAP` above? That will increase the test coverage for these distributions.